packages/suite/src/components/suite/layouts/WelcomeLayout/WelcomeSidebar.tsx

```diff
@@ -0,0 +1,155 @@
+import { TorContainerWithCheckIcon } from '../SuiteLayout/Sidebar/TorContainerWithCheckIcon';
+
+type ActivePage = 'trezor' | 'settings';
+
+const WELCOME_SIDEBAR_WIDTH = 84;
+const WELCOME_ICON_SIZE = 40;
+
+const WelcomeWrapper = styled.div`
+    background-color: ${({ theme }) => theme.backgroundSurfaceElevationNegative};
+    border-right: 1px solid ${({ theme }) => theme.borderElevation0};
+
+    @media (max-width: ${variables.SCREEN_SIZE.MD}) {
+        display: none;
+    }
+`;
+
+const LeftContainer = styled.div`
+    padding: ${spacingsPx.xs};
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 100%;
+`;
+
+const LeftWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: ${spacingsPx.xs};
+`;
+
+type ExtraIconButtonProps = {
+    variant: ButtonVariant;
+    borderRadius: string;
+    width: number;
+    height: number;
+};
+
+const extraIconButtonProps = (): ExtraIconButtonProps => {
+    return {
+        variant: 'welcome',
+        borderRadius: borders.radii.sm,
+        width: WELCOME_ICON_SIZE,
+        height: WELCOME_ICON_SIZE,
+    };
+};
+
+export const WelcomeSidebar = () => {
+    const dispatch = useDispatch();
+    const { translationString } = useTranslation();
+    const isDiscreetModeActive = useSelector(selectIsDiscreteModeActive);
+    const translationLabel = isDiscreetModeActive ? 'TR_SHOW_BALANCES' : 'TR_HIDE_BALANCES';
+    const { isTorLoading } = useSelector(selectTorState);
+    const [activePage, setActivePage] = useState<ActivePage>('trezor');
+    const location = useLocation();
+
+    const handleDiscreetModeClick = () => dispatch(setDiscreetMode(!isDiscreetModeActive));
+
+    const handleClickTrezor = () => {
+        setActivePage('trezor');
+        dispatch(goto('suite-start'));
+        const updatedUrl = removeSettingsFromUrl(window.location.pathname);
+        window.location.pathname = updatedUrl;
+        location.pathname = removeSettingsFromUrl(location.pathname);
+    };
+
+    const handleClickSettings = useCallback(() => {
+        setActivePage('settings');
+        dispatch(goto('settings-index'));
+    }, [dispatch]);
+
+    const handleClickTor = () => {
+        setActivePage('settings');
+        dispatch(
+            goto('settings-index', {
+                anchor: SettingsAnchor.Tor,
+            }),
+        );
+    };
+
+    useEffect(() => {
+        if (
+            window.location.pathname.includes('settings') ||
+            location.pathname.includes('settings')
+        ) {
+            setActivePage('settings');
+        }
+    }, [location.pathname]);
+
+    return (
+        <WelcomeWrapper>
+            <TrafficLightOffset>
+                <Column
+                    justifyContent="center"
+                    alignItems="center"
+                    height="100%"
+                    width={WELCOME_SIDEBAR_WIDTH}
+                >
+                    <LeftContainer>
```

Comment:
you are using `Column` here probably only for centering content but at the same time you use `LeftContainer` which is almost the same equivalent. Could you please merge it? Ideally to `Column`